packages/utils/jq/build.yaml

```diff
@@ -0,0 +1,13 @@
+requires:
+- name: "base"
+  category: "distro"
+  version: ">=0"
+
+package_dir: /jq
+
+steps:
+- | 
+   PACKAGE_VERSION=${PACKAGE_VERSION%\+*} && \
+   mkdir -p /${PACKAGE_NAME}/usr/bin && \
+   curl -L https://github.com/stedolan/jq/releases/download/jq-${PACKAGE_VERSION}/jq-linux64 -o /${PACKAGE_NAME}/usr/bin/${PACKAGE_NAME} && \
```

Comment:
can we use `labels.github.owner` and `labels.github.repo` here? I found it pretty useful to test pre-release or patched releases from my own forks

Reply:
done!